src/app/core/admin/admin-entity-details/widget-component-select/widget-component-select.component.ts

```diff
@@ -89,3 +184,9 @@ export interface WidgetOption {
    */
   disabled?: string;
 }
+
+export interface DashboardWidgetOption {
+  label: string;
+  value: DynamicComponentConfig;
+  disabled?: string;
+}
```

Comment:
I think this is structurally exactly the same as `WidgetOption`? Let's use the existing interface then